packages/core/src/types/IRetrieveConfiguration.ts

```diff
@@ -0,0 +1,45 @@
+import { RequestType } from '../enums';
+
+export interface RetrieveStage {
+  id: string;
+  // Set of positions - negative values are relative to the end, positive to
+  // the beginning, and fractional values between 0 and 1 are relative to frame count
+  positions?: number[];
+  // Alternately, choose positions by decimating at the given interval
+  decimate?: number;
+  // With the given offset to decimation
+  offset?: number;
+  // Use a specified retrieve type to add fetch arguments and configuration
+  // on the retrieve URL.
+  retrieveTypeId?: string;
+  // The type of request to use
+  requestType?: RequestType;
+  // THe priority to use
+  priority?: number;
+}
+
+export interface LossyConfiguration {
```

Comment:
I firmly believe that `ImageRetrievalConfiguration` may be a more accurate representation of the information it contains.